react/components/organisms/ripe-configurator/ripe-configurator.js

```diff
@@ -321,9 +326,10 @@ export class RipeConfigurator extends mix(Component).with(LogicMixin) {
 
     async _updateConfiguration(props, prevProps) {
         if (
-            prevProps.brand !== props.brand ||
-            prevProps.model !== props.model ||
-            prevProps.version !== props.version
+            props.config &&
+            (prevProps.brand !== props.brand ||
+                prevProps.model !== props.model ||
+                prevProps.version !== props.version)

```

Comment:
Add this as a separate `if` inside the body of the `true` clause. The reason for that is: unlike what happens with Vue, in React you need to check if the prop really changed in order to have watchers that only trigger on changes. Because of that, and because we want to keep the implementation as similar as possible to Vue, the equivalent of copying the watch handler inside here would be having the if inside.
```suggestion
        (prevProps.brand !== props.brand ||
                prevProps.model !== props.model ||
                prevProps.version !== props.version) {
                if (props.config) await ...

```